projects/bactteria_detection/README.md

```diff
@@ -0,0 +1,116 @@
+# Bactteria detection with darkfield microscopy
+
+## Description
+
+This project support **`Bactteria detection with darkfield microscopy `**, and the dataset used in this project can be downloaded from [here](https://tianchi.aliyun.com/dataset/94411).
+
+### Prerequisites
+
+- Python 3.8
+- PyTorch 1.10.0
+- pillow(PIL)
+- scikit-learn(sklearn)
+- [MIM](https://github.com/open-mmlab/mim) v0.3.4
+- [MMCV](https://github.com/open-mmlab/mmcv) v2.0.0rc4
+- [MMEngine](https://github.com/open-mmlab/mmengine) v0.2.0 or higher
+- [MMSegmentation](https://github.com/open-mmlab/mmsegmentation) v1.0.0rc5
+
+### Dataset preparing
+
+- download dataset from [here](https://tianchi.aliyun.com/dataset/94411) and decompression data to path 'data/bactteria_detection'.
+- run script `"python projects/bactteria_detection/tools/prepare_dataset.py"` to split dataset and change folder structure as below.
+
+```none
+  mmsegmentation
+  ├── mmseg
```

Comment:
as the root is ` bactteria_detection/` and the `data_root` is config file is relative path `'data/bactteria_detection'`, I think the the structure of project and dataset should be like
```suggestion
 bactteria_detection
  ├── datasets
```